### DISCUSSION TITLE
Github Sponsoring?

### DISCUSSION THREAD

**Discussion by crubier** [chakra-ui/chakra-ui#3233] [General]
Hi @segunadebayo ,

I'd like to sponsor Chakra with my company, but it would be much easier for us if we could pay through Github sponsor, for accountability, taxes, etc.

Is that something you though of?

**Comment by segunadebayo** [comment]
Hi Vincent @crubier,

We're in the process of setting up a business bank account to allow us to enable Github Sponsor. Are you able to use Open Collective instead I think they might be able to provide similar accountability & taxes?

We'd love you have you as a sponsor. Let me know
